docs/running-workflows/launching-workflows/index.md

```diff
@@ -0,0 +1,125 @@
+
+```text
+https://reana.cern.ch/launch?name=my-new-name&url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+### specification (optional)
+
+If REANA specification file is different from the default `reana.yaml`,
+you can specify the path to the desired specification file using `specification` argument:
+
+```text
+https://reana.cern.ch/launch?specification=reana-snakemake.yaml?url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+### parameters (optional)
+
+REANA specification file allows defining parameters for the workflow.
+In some cases, you may want to change the default values of these parameters.
+
+For example, [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit)
+has `events` parameter in the specification file. By default, it is `20000`. Let's reduce this parameter to `10000`:
+
+```text
+https://reana.cern.ch/launch?parameters={"events": "10000"}&url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+## Launcher badges
+
+In order to facilitate launching your analysis workflows directly from GitHub or GitLab, you may want to generate a badge pointing to the launcher URL with the desired arguments and then include this badge in your analysis documentation.
+
+The standard badge looks like this:
+
+![Launch on REANA](https://www.reana.io/static/img/badges/launch-on-reana.svg)
+![Launch on REANA at CERN](https://www.reana.io/static/img/badges/launch-on-reana-at-cern.svg)
+![Launch with CWL on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-cwl-on-reana-at-cern.svg)
+![Launch with Yadage on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-yadage-on-reana-at-cern.svg)
+![Launch with Serial on REANA at CERN](https://www.reana.io/static/img/badges/launch-with-serial-on-reana-at-cern.svg)
+![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
+
+### Markdown
+
+Badge can be added in Markdown using the following syntax:
+
+```markdown
+[![alt text](https://www.reana.io/static/img/badges/launch-on-reana.svg)](<WORKFLOW_LAUNCH_URL>)
+```
+
+Substitute your URL instead of `WORKFLOW_LAUNCH_URL`:
+
+```markdown
+[![Launch RooFit demo workflow from GitHub on REANA](https://www.reana.io/static/img/badges/launch-on-reana.svg)](https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit)
+```
+
+### HTML
+
+Badge can be added in HTML using the following syntax:
+
+```html
+<a href="<WORKFLOW_LAUNCH_URL>"><img src="https://www.reana.io/static/img/badges/launch-on-reana.svg" alt="Launch RooFit demo workflow from GitHub on REANA"></a>
+```
+
+Substitute your URL instead of `WORKFLOW_LAUNCH_URL`:
+
+```html
+<a href="https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit"><img src="https://www.reana.io/static/img/badges/launch-on-reana.svg" alt="Launch RooFit demo workflow from GitHub on REANA"></a>
+```
```

Comment:
What about adding a section on how to generate custom badges using `shields.io` or `badget.net`? Could be a simple:

If you would like to generate your own custom badge, you can use external services such as `shields.io` or `badgen.net`.

(if we don't present an example, this phrase could even go after our phrase above about the standard set of badges)

Reply:
I added one sentence at the end. We can extend this section in the next PR